tests/unit/spam_user.t

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/perl -w
+
+use ProductOpener::PerlStandards;
+
+use Test::More;
+use Log::Any::Adapter 'TAP';
+
+use ProductOpener::Users qw/:all/;
+
+foreach my $name ("click here", "go forms.yandex.ru", "https://test.me") {
+	ok(is_supsicious_name($name), "$name is a suspicious name");
+}
+
+foreach my $name ("danilo.go", 'roberto@test.me', "Lalilou", "Dan stanley") {
```

Comment:
Currently email addresses such as roberto@test.me should trigger the suspicious detection, no?

Reply:
For me it's another problem. I mean for an email we should return a real error to the user (not a cryptic one)